server.js

```diff
@@ -20,4 +24,80 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+// returining all messages
+app.get("/messages", (req, res) => {
+  const queryText = req.query.text;
+  messages.forEach((message) => console.log(message.text.includes(queryText)));
+
+  // console.log(messages.filter((message) => message.text.includes(queryText)));
+  if (queryText) {
+    return res.json(
+      messages.filter((message) =>
+        message.text.toLowerCase().includes(queryText.toLowerCase())
+      )
+    );
+  }
+
+  res.json(messages).send;
```

Comment:
`send` is a method on the object returned from `res.json` - you should either call it here with a message or omit it